src/test/rustdoc-gui/sidebar.goml

```diff
@@ -1,8 +1,10 @@
 goto: file://|DOC_PATH|/test_docs/index.html
+show-text: true
 assert-text: (".sidebar > .location", "Crate test_docs")
 // In modules, we only have one "location" element.
 assert-count: (".sidebar .location", 1)
-assert-text: (".sidebar-elems > #all-types", "See all test_docs's items")
+assert-text: ("#all-types", "All Items")
+assert-css: ("#all-types", {"color": "rgb(56, 115, 173)"})
```

Comment:
When you show text to check color, please set the theme as it might fail if someone has a different one on their computer.

Reply:
Done! Note: the GUI tests should probably set the theme explicitly to light by default so they are never impacted by settings on the computer they're running on.

Reply:
I thought they were impacted by the system theme settings. Maybe it was another bug. In any case, thanks for going along with it!